src/os/ObjectStore.cc

```diff
@@ -20,9 +20,7 @@
 
 #include "filestore/FileStore.h"
 #include "memstore/MemStore.h"
-#if defined(WITH_BLUESTORE)
```

Comment:
this practically breaks the build on FreeBSD, where bluestore's aio backend is not ready yet.

Reply:
@tchaikov Thanks for your comment. That has given me more task. I can't imagine how wonderful `BlueStore` becomes default `ObjectStore` on all supported platform by us!